elated-egrets/bangazon-internal-site-2.0
# Finished file structure for folders and split off certain files

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout JRfolder

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. check file structure and init files
1. make sure that the imports are correct 

## What you changed
file structure and location of models, forms, and views
## What was added to read me
N/A
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
